packages/playground/remote/src/lib/worker-caching.ts

```diff
@@ -0,0 +1,96 @@
+import { isURLScoped } from '@php-wasm/scopes';
+
+export class WorkerCache {
```

Comment:
Nit: let's name this after why it's useful, not where it's used e.g. PlaygroundOfflineModeCache

Reply:
I renamed it to `OfflineModeCache`.